### ISSUE TITLE
fail on boost 1.59 (Trac #1467)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1459]
cmake checks for boost 1.58 and fails if detected, newer version of boost such as 1.59 don't work either and should have the same error message

Migrated from https://code.icecube.wisc.edu/ticket/1467
```json
{
    "status": "closed", 
    "changetime": "2019-01-12T00:05:34", 
    "description": "cmake checks for boost 1.58 and fails if detected, newer version of boost such as 1.59 don't work either and should have the same error message", 
    "reporter": "kjmeagher", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547251534526345", 
    "component": "cmake", 
    "summary": "fail on boost 1.59", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-12-07T10:22:19", 
    "milestone": "", 
    "owner": "nega", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20151207T14:36:33: 
* nega changed status from "new" to "closed"
* nega changed owner from "" to "nega"
* nega commented:

> In changeset 2360:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2360"
> trivial adjustment for > boost 1.58
> 
> closes #1459
> ```


* nega changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190112T00:05:34: nwhitehorn commented:

> In changeset 170155:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170155"
> trivial adjustment for > boost 1.58
> 
> closes #1459
> ```
